Block/Analytics.php

```diff
@@ -74,4 +86,38 @@ protected function _toHtml()
         }
         return parent::_toHtml();
     }
+
+    /**
+     * Track page views
+     *
+     * @return mixed
+     */
+    public function getEvent()
+    {
+        if (!$this->fullActionName || $this->isRejected($this->fullActionName)) {
+            return;
+        }
+        
+        switch($this->fullActionName) {
+            // product view pages
+            case 'catalog_product_view':
+                return $this->productViewEvent;
+            default:
+                break;
+        }
+    }
+
+    /**
+     * Events that we don't want to track
+     *
+     * @param string full action name
+     */
+    protected function isRejected($action)
+    {
+        $rejected = [
+            'catalogsearch_advanced_index',
```

Comment:
Add a story to think about these. Why are they being ignored?

Reply:
Story url: https://metrilojira.atlassian.net/browse/PLG-110